src/ui/digital-exchange/MarketplaceFilter.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm } from 'redux-form';
+import CheckboxGroup from 'ui/digital-exchange/common/CheckboxGroup';
+import SidebarFilter from 'ui/digital-exchange/common/SidebarFilter';
+
+class MarketplaceFilterBody extends Component {
+  componentWillMount() {
+    this.props.onWillMount();
+  }
+
+  render() {
+    const options = this.props.digitalExchangeMarketplaces.map(marketplace => (
+      { label: marketplace, value: marketplace }
+    ));
+
+    return (
+      <SidebarFilter title="Marketplace">
```

Comment:
I'm afraid you will also get an unwanted tooltip using `title` as prop name. Do we need that?

Reply:
@gidesan I don't see a tooltip, where do you see it?

Reply:
if no tooltip that's fine, I can resolve the conversation